custom_components/waste_collection_schedule/waste_collection_schedule/source/frankston_vic_gov_au.py

```diff
@@ -0,0 +1,136 @@
+import requests
+import json
+from shapely.geometry import Point, MultiPolygon, Polygon
```

Comment:
shapely currently is not a dependency. We tried adding shapely as dependency in the past and some people had problems running this integration at all (#1705)

Reply:
Ah bugger. Thanks, I'll have a go at replicating the behaviour without Shapely.

Reply:
Resolved with https://github.com/mampfes/hacs_waste_collection_schedule/pull/2141/commits/0205c5dfa4aea9aefbd481c73bfd9cc205a630f6